### PR TITLE
[FIX] Bug Fixes Related to Installation

### PR DESCRIPTION
**Pull request by Tarun-Arora**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [X] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that another pull request for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [X] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

This PR fixes the issue mentioned in #606

### CODE REVIEW

**Review by thealphadollar — DISMISSED** [review]
LGTM.

**Comment by Tarun-Arora** [comment]
> LGTM.

@thealphadollar Sir I saw that build failed due to some issue in ```./mod_test/models.py```. I think I have fixed it now. Please review.

**Review by thealphadollar — APPROVED** [review]

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Please fix the tests.

File: mod_auth/forms.py [hunk 1]
```diff
@@ -5,8 +5,7 @@
 
 from flask_wtf import FlaskForm
 from wtforms import PasswordField, SelectField, StringField, SubmitField
-from wtforms.fields.html5 import EmailField
-from wtforms.fields.simple import PasswordField
+from wtforms.fields import EmailField, PasswordField
```
Comment (canihavesomecoffee) [inline mod_auth/forms.py:8]: Please revert this change, it's unnecessary. See #616

File: install/install.sh [hunk 2]
```diff
@@ -121,7 +121,7 @@ read -e -r -p "GitHub Owner Name : " -i "CCExtractor" github_owner_name
 read -e -r -p "GitHub repository : " -i "ccextractor" github_repository
 read -e -r -p "Email Domain : " -i "${config_server_name}" email_domain
 read -e -r -p "Email API key (Generate one here https://www.mailgun.com/) : " -i "" email_api_key
-hmac_key=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+hmac_key=$(head -80 /dev/urandom | LC_ALL=c tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```
Comment (canihavesomecoffee) [inline install/install.sh:124]: Can you explain what issues you ran into that caused you to change this?

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

File: mod_auth/forms.py (hunk 1 above)
Comment (Tarun-Arora) [reply mod_auth/forms.py:8]: Okay Sir, will remove this now.

File: install/install.sh (hunk 2 above)
Comment (Tarun-Arora) [reply install/install.sh:124]: Running of this command ```< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1``` prints the hmac_key in the console, and this does not terminate until one manually cease it, due to this, the installation gets hung at this step (it did at least in my local computer).

**Comment by canihavesomecoffee** [comment]
Can you re-base the branch on current main so there's only a single commit with the proposed fix?

File: install/install.sh (hunk 2 above)
Comment (thealphadollar) [reply install/install.sh:124]: I can confirm the new command works on my system. The old one seems to be giving error.

**Review by thealphadollar — DISMISSED** [review]
Thanks for the work, Tarun.

Please rebase so that there are changes only in a single file.

**Review by canihavesomecoffee — APPROVED** [review]

**Comment by Tarun-Arora** [comment]
> Thanks for the work, Tarun.
> 
> Please rebase so that there are changes only in a single file.

Succesfully did rebase and also fetched the upstream.
